schema/5.1.0/schema.md

```diff
@@ -1221,7 +1221,7 @@ To display a dataset in CELLxGENE Explorer, Curators MUST annotate **one or more
       <td>X_{suffix} with the following requirements:<br><br>
       <ul>
         <li>{suffix} MUST be at least one character in length.</li>
-        <li>The first character of {suffix} MUST be a letter of the alphabet and the remaining characters MUST be alphanumeric characters. (This is equivalent to the regular expression pattern <code>"^[a-zA-Z][a-zA-Z0-9]*$"</code>.)</li>
+        <li>The first character of {suffix} MUST be a letter of the alphabet and the remaining characters MUST be alphanumeric characters or one of: `_`, `-`, `.` (This is equivalent to the regular expression pattern <code>"^[a-zA-Z][a-zA-Z0-9_.-]*$"</code>.)</li>
```

Comment:
The schema use a mixture of HTML and markdown. The HTML is used for tables.

As a result, you can't use ` in a HTML section. (View the file to see how it's rendered)

Replace:


```
one of: `_`, `-`, `.`
```

with:

```
one of <code>'_'</code>,<code>'-'</code>,<code>'.'</code>
```